.gitignore

```diff
@@ -29,4 +29,18 @@ yarn-error.log*
 
 # visual studio folder
 /.vs
-/.scannerwork
\ No newline at end of file
+/.scannerwork
+
+# Emacs temporary files
+\#*\#
+*~
+.\#*
+
+# Auto-save files
+*#
```

Comment:
Now correct me if I am wrong, but don't you need to escape these pounds, because I think what this does is gitignore everything in the base of the repo